resources/fixtures/Eurolite-KLS200.qxf

```diff
@@ -23,86 +23,87 @@
   <Colour>Green</Colour>
-  <Capability Min="0" Max="255">Spot 1: Green</Capability>
+  <Capability Min="0" Max="255">Dimming from 0 - 100%</Capability>
  </Channel>
  <Channel Name="Spot 1: Blue">
   <Group Byte="0">Intensity</Group>
   <Colour>Blue</Colour>
-  <Capability Min="0" Max="255">Spot 1: Green</Capability>
+  <Capability Min="0" Max="255">Dimming from 0 - 100%</Capability>
  </Channel>
  <Channel Name="Spot 2: Red">
   <Group Byte="0">Intensity</Group>
   <Colour>Red</Colour>
-  <Capability Min="0" Max="255">Spot 2: Red</Capability>
+  <Capability Min="0" Max="255">Dimming from 0 - 100%</Capability>
  </Channel>
  <Channel Name="Spot 2: Green">
   <Group Byte="0">Intensity</Group>
   <Colour>Green</Colour>
-  <Capability Min="0" Max="255">Spot 2: Green</Capability>
+  <Capability Min="0" Max="255">Dimming from 0 - 100%</Capability>
  </Channel>
  <Channel Name="Spot 2: Blue">
   <Group Byte="0">Intensity</Group>
   <Colour>Blue</Colour>
-  <Capability Min="0" Max="255">Spot 2: Blue</Capability>
+  <Capability Min="0" Max="255">Dimming from 0 - 100%</Capability>
  </Channel>
  <Channel Name="Spot 3: Red">
   <Group Byte="0">Intensity</Group>
   <Colour>Red</Colour>
-  <Capability Min="0" Max="255">Spot 3: Red</Capability>
+  <Capability Min="0" Max="255">Dimming from 0 - 100%</Capability>
  </Channel>
  <Channel Name="Spot 3: Green">
   <Group Byte="0">Intensity</Group>
   <Colour>Green</Colour>
-  <Capability Min="0" Max="255">Spot 3: Green</Capability>
+  <Capability Min="0" Max="255">Dimming from 0 - 100%</Capability>
  </Channel>
  <Channel Name="Spot 3: Blue">
   <Group Byte="0">Intensity</Group>
   <Colour>Blue</Colour>
-  <Capability Min="0" Max="255">Spot 3: Blue</Capability>
+  <Capability Min="0" Max="255">Dimming from 0 - 100%</Capability>
  </Channel>
  <Channel Name="Spot 4: Red">
   <Group Byte="0">Intensity</Group>
   <Colour>Red</Colour>
-  <Capability Min="0" Max="255">Spot 4: Red</Capability>
+  <Capability Min="0" Max="255">Dimming from 0 - 100%</Capability>
  </Channel>
  <Channel Name="Spot 4: Green">
   <Group Byte="0">Intensity</Group>
   <Colour>Green</Colour>
-  <Capability Min="0" Max="255">Spot 4: Green</Capability>
+  <Capability Min="0" Max="255">Dimming from 0 - 100%</Capability>
  </Channel>
  <Channel Name="Spot 4: Blue">
   <Group Byte="0">Intensity</Group>
   <Colour>Blue</Colour>
-  <Capability Min="0" Max="255">Spot 4: Blue</Capability>
+  <Capability Min="0" Max="255">Dimming from 0 - 100%</Capability>
  </Channel>
  <Mode Name="Mode 1">
   <Physical>
    <Bulb Lumens="0" Type="LED" ColourTemperature="0"/>
-   <Dimensions Width="0" Height="0" Weight="3" Depth="0"/>
-   <Lens DegreesMax="0" Name="PC" DegreesMin="0"/>
-   <Focus PanMax="0" Type="Fixed" TiltMax="0"/>
-   <Technical PowerConsumption="70" DmxConnector="3-pin"/>
+   <Dimensions Width="1110" Depth="59" Weight="3" Height="295"/>
+   <Lens DegreesMin="25" DegreesMax="25" Name="Other"/>
+   <Focus Type="25" TiltMax="0" PanMax="0"/>
```

Comment:
The tab order on this form could do with changing. It goes left-right instead of up-down, hence my mistake.